src/wasi/Wasi.cpp

```diff
@@ -31,17 +35,20 @@ WASI::WasiFunc* WASI::find(std::string funcName)
 void WASI::test(ExecutionState& state, Value* argv, Value* result, void* data)
 {
     printf("No argument test succesful.\n");
+    result[0] = Value(static_cast<uint16_t>(WASI::wasi_errno::success));
 }
 
 void WASI::printI32(ExecutionState& state, Value* argv, Value* result, void* data)
 {
     printf("Recieved number: %d.\n", argv[0].asI32());
+    result[0] = Value(static_cast<uint16_t>(WASI::wasi_errno::success));
```

Comment:
I cannot find error handling from the document you mentioned (https://github.com/WebAssembly/WASI/blob/main/legacy/preview1/docs.md)
To be specific, I found `errno: Variant` but could find `test, printI32, and writeI32` functions.
Are these functions standard WASI features?

Reply:
No, these were just used for testing to see if arguments were handled right. They could be removed without any issues.

Reply:
Please remove all non-standard WASI functions.
I think it's better to implement and test standard features only.

Reply:
I have removed the non-standard functions.